posts/month-in-mathlib-nov-2021.md

```diff
@@ -0,0 +1,84 @@
+  - [PR #9995](https://github.com/leanprover-community/mathlib/pull/9995) diagonalization of self-adjoint endomorphisms (finite dimension)
+  - [PR #10317](https://github.com/leanprover-community/mathlib/pull/10317) more concrete diagonalization theorem
+  - [PR #10335](https://github.com/leanprover-community/mathlib/pull/10335) Continuous affine maps are smooth
+
+* Convergence.
+  - [PR #10073](https://github.com/leanprover-community/mathlib/pull/10073) Convergence of a sequence which does not oscillate infinitely
+  - [PR #10258](https://github.com/leanprover-community/mathlib/pull/10258) Prove that, if u_n is subadditive, then u_n / n converges.
+
+* Seminorms
+  - [PR #9097](https://github.com/leanprover-community/mathlib/pull/9097) Define the Minkowski functional
+  - [PR #10203](https://github.com/leanprover-community/mathlib/pull/10203) A seminorm is convex
+
+* Homotopy theory.
+  - [PR #9865](https://github.com/leanprover-community/mathlib/pull/9865) Add `homotopic` for `continuous_map`s.
+  - [PR #10195](https://github.com/leanprover-community/mathlib/pull/10195) The functor from `Top` to `Groupoid`
+
+* Number theory.
+  - [PR #9071](https://github.com/leanprover-community/mathlib/pull/9071) Define the class number of number fields and function fields
+  This finishes the finiteness proof of the class group of a number field and function field, described in [a previous blog post](https://leanprover-community.github.io/blog/posts/dedekind-domains-and-class-number-in-lean/).
+  - [PR #8820](https://github.com/leanprover-community/mathlib/pull/8820) Add theorem for Lucas primality test
+
+* Group theory.
+  - [PR #10059](https://github.com/leanprover-community/mathlib/pull/10059) (Infinite) Sylow subgroups are isomorphic
+  - [PR #10283](https://github.com/leanprover-community/mathlib/pull/10283) Prove the full Schur-Zassenhaus theorem
+  - [PR #10249](https://github.com/leanprover-community/mathlib/pull/10249) Define the exponent of a group
+
+* Sheafification.
+  An important building block in setting up for algebraic geometry,
+  as well as a requirement for the Liquid Tensor Experiment.
+  - [PR #10284](https://github.com/leanprover-community/mathlib/pull/10284) `P⁺` for a presheaf `P`.
+  - [PR #10297](https://github.com/leanprover-community/mathlib/pull/10297) If `P` is a sheaf, then the map from `P` to `P⁺` is an isomorphism.
+  - [PR #10303](https://github.com/leanprover-community/mathlib/pull/10303) The sheafification of a presheaf.
+  - [PR #10328](https://github.com/leanprover-community/mathlib/pull/10328) `Sheaf_to_presheaf` creates limits
+  - [PR #10334](https://github.com/leanprover-community/mathlib/pull/10334) `Sheaf J D` has colimits.
+  - [PR #10401](https://github.com/leanprover-community/mathlib/pull/10401) `SheafedSpace` has colimits
+  A side product is the important corollary that the category of sheaves
+  with values in a suitable concrete abelian category is itself an abelian category.
```

Comment:
This is not completely accurate -- more work is required to obtain this result (e.g. the fact that sheafification is left exact, and a more general version of the fact that filtered colimits commute with finite limits) -- both of these have been done in LTE, but not yet in mathlib.

Reply:
Hmm, that's right. I will reword.